app/src/main/java/org/schabi/newpipe/local/feed/service/FeedLoadService.kt

```diff
@@ -345,6 +347,17 @@ class FeedLoadService : Service() {
                     error is IOException -> throw error
                     cause is IOException -> throw cause
                     error.isNetworkRelated -> throw IOException(error)
+
+                    cause is ContentNotAvailableException -> {
+                        // maybeWrapper is definitely a RequestException,
+                        // because this is an exception thrown in the extractor
+                        if (maybeWrapper is RequestException) {
+                            Log.d(TAG, if (error.message == null) "terminated" else error.message)
```

Comment:
```suggestion
                            Log.d(TAG, error.message ?: "terminated")
```

Reply:
Removed. I did not want to add that debug message